source/2019-04-05-the-ember-times-issue-92.md

```diff
@@ -29,56 +33,63 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## ["I contribute to Ember" with Chris Garrett 💬](https://discuss.emberjs.com/t/i-contribute-to-ember-with-chris-garrett/16395)
+
+<div class="float-right padded portrait-frame">
+  <img alt="Chris Garrett" title="Chris Garrett - Contributor to Ember" src="/images/blog/emberjstimes/pzuraq.jpg" />
+</div>
+
+In the 9th edition of the contributor interview series, community member **Chris Garrett**, also known as [@pzuraq](https://github.com/pzuraq) talks about his story getting involved with working on **native classes** and **decorators** in Ember and why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```

Comment:
Looks great!!!

I would consider adding a comma after the reference to the pzuraq handle and link or moving the comma trailing Chris Garrett to after the handle and link and putting that and the `also known as` in parens.

Reply:
```suggestion
In the 9th edition of the contributor interview series, community member **Chris Garrett**, also known as [@pzuraq](https://github.com/pzuraq), talks about his story getting involved with working on **native classes** and **decorators** in Ember and why everyone in the community is eager to help **make your contribution** a real **success** as well ✨
```